docs/quick-start.md

```diff
@@ -88,9 +88,10 @@ edit:add-var y~ {|@argv|
 ```powershell
 function y {
     $tmp = [System.IO.Path]::GetTempFileName()
-    yazi $args --cwd-file="$tmp"
```

Comment:
Please keep the `$args` here otherwise we can't pass arguments to Yazi by using the shell wrapper like `y --debug`

Reply:
Added back in latest commit.